Assets/Scripts/LightBulbController.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class LightBulbController : MonoBehaviour
+{
+
+    public bool lightOn = false;
```

Comment:
Class variables should be initialized in Prefab.prefab according to previous lecture if I'm not mistaken

Reply:
not necessarily but it has been changed